### PR TITLE
Add Spanish Support

### PR DESCRIPTION
**Pull request by carlosthe19916**

### CODE REVIEW

**Comment by wjur** [comment]
Hi @carlosthe19916 and @costris95! I'm sorry for the huge delay. There is another PR (#16) with Spanish, but it is not finished, too. Is any of you still willing to contribute?

**Comment by israteneda** [comment]
There is an update on this pull request. Is any change necessary?

File: src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/spanish/SpanishValuesTest.groovy [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package pl.allegro.finance.tradukisto.internal.languages.spanish
+
+import spock.lang.Specification
+
+import static pl.allegro.finance.tradukisto.internal.languages.GenderForms.genderForm
+
+class SpanishValuesTest extends Specification {
+
```
Comment (jglaszka) [inline src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/spanish/SpanishValuesTest.groovy:8]: Hi, can you add more test cases? You can get inspired by `BrazilianPortugueseValuesTest` and fill with spanish translations.

**Review by jglaszka — CHANGES_REQUESTED** [review]
I would like to see more unit tests.

**Comment by carlosthe19916** [comment]
Hey, this PR got lost from my radar the last 5 years. I'm no longer interested in continuing with this PR. I'm happy to close it without merging it.

**Comment by jglaszka** [comment]
> Hey, this PR got lost from my radar the last 5 years. I'm no longer interested in continuing with this PR. I'm happy to close it without merging it.

Okay, I understand. Sorry for the delay. This project was neglected by the past owners, but currently I started to clean it up and take care of the project. 
If I will apply some changes, can I merge your code in future?

**Comment by carlosthe19916** [comment]
> > Hey, this PR got lost from my radar the last 5 years. I'm no longer interested in continuing with this PR. I'm happy to close it without merging it.
> 
> Okay, I understand. Sorry for the delay. This project was neglected by the past owners, but currently I started to clean it up and take care of the project. If I will apply some changes, can I merge your code in future?

Sure, you can use the code in this PR as you need.

**Comment by jglaszka** [comment]
Thank you! 😊

**Comment by GHim9182** [comment]
Hi @jglaszka, I need the Spanish support for my company. After I implemented Italian and Dutch conversions I would love to help also in the Spanish one.
I see there are different PR, could you kindly update me on the status of the ES support and in case, which  branch I should use?

Thanks.

**Comment by GHim9182** [comment]
@jglaszka sorry, just noticed it's now under development and review in #111 . nvm

**Comment by jglaszka** [comment]
@GHim9182 Support for spanish was released in https://github.com/allegro/tradukisto/releases/tag/v2.6.0

**Pull request closed without merging**